### PR TITLE
Adds display component for widgets with no content

### PR DESCRIPTION
**Pull request by Zachary-Dernar**
Also modifies the message found in the No Results page
to reflect the wording in the No Content component.

### CODE REVIEW

File: views/NoContent/index.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+import { Icon } from 'watson-react-components';
+
+const NoContent = (props) => (
+  <div className="no-content--wrapper">
+    <div className="no-content--icon">
+      <Icon style={{ width: '3.7rem', height: '3.7rem', fill: '#00bfb0' }} type="info" />
```
Comment (maniax89) [inline views/NoContent/index.jsx]: try using css `.no-content--icon .icon` for your CSS

```
.no-content--icon .icon {
  width: 3.7rem;
  height: 3.7rem;
  fill: #00bfb0;
}
```

File: views/NoContent/index.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        "Try expanding the timeframe."
+      ) : (
+        "Try modifying your query."
+      )}
+    </p>
+    </div>
+  </div>
+);
+
+// <p className={props.query.restrictedDateRange ? 'no-content--timeframe' : 'hidden'}>
```
Comment (maniax89) [inline views/NoContent/index.jsx]: remove this

**Review by maniax89 — COMMENTED** [review]
add a test for both

- Mentions And Sentiments
- Top Entities

for when <NoContent /> is shown

File: test/views/TopEntities/test.index.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+//import NoContent from '../../../views/NoContent/index.jsx';
```
Comment (maniax89) [inline test/views/TopEntities/test.index.js]: remove this

**Review by kognate — APPROVED** [review]
LGTM

**Pull request merged**